docs/developer-docs/integrations/ethereum/evm-rpc.md

```diff
@@ -0,0 +1,365 @@
+# EVM RPC canister
+
+To use the EVM RPC canister, you can import it into your project's `dfx.json` file:
+
+```json
+{
+ "canisters": {
+   "evm_rpc": {
+     "type": "custom",
+     "candid": "https://github.com/internet-computer-protocol/ic-eth-rpc/releases/latest/download/evm_rpc.did",
+     "wasm": "https://github.com/internet-computer-protocol/ic-eth-rpc/releases/latest/download/evm_rpc_dev.wasm.gz",
+     "remote": {
+       "id": {
+         "ic": "a6d44-nyaaa-aaaap-abp7q-cai"
+       }
+     }
+   }
+ }
+}
+```
+
+## Supported RPC methods
+
+- `eth_gasPrice`: Queries the current ETH gas price.
+- `eth_getLogs`: Queries the logs of a specified block or transaction.
+- `eth_getBlockByNumber`: Queries information about a given block.
+- `eth_getTransactionReceipt`: Queries details about a submitted transaction.
+- `eth_feeHistory`: Queries the historical fee data to estimate gas prices for transactions.
+- `eth_sendRawTransaction`: Submits a signed transaction to the Ethereum network.
+- `eth_getTransactionCount`: Queries the number of transactions for a specified address.
+- `eth_call`: Calls a smart contract function without changing the blockchain's state.
+- `eth_getStorageAt`: Queries the value of a storage position at a given address.
+- `eth_estimateGas`: Provides an estimate for the gas consumption of a transaction before it is mined.
+- `eth_blockNumber`: Queries the number of the most recent block.
+- `eth_getCode`: Queries the bytecode of a specified contract.
+
+
+## Supported RPC providers
+
+- Alchemy: `https://eth-mainnet.g.alchemy.com/v2/`
+- Ankr: `https://rpc.ankr.com/eth/`
+- BlockPI: `https://ethereum.blockpi.network/v1/rpc/`
+- Cloudflare Web3: `https://cloudflare-eth.com/v1/mainnet/`
+- Public Node: `https://ethereum.publicnode.com`
+
+View the [full list of RPC providers for each EVM network.](https://chainlist.org/)
+
+### Specifying an RPC provider
+
+To specify the RPC provider you'd like to use, change the `Url` value:
```

Comment:
It's also possible to specify RPC providers in other ways. Here's the Candid interface:

```candid
type JsonRpcSource = variant {
  Chain : nat64;
  Service : record { hostname : text; chainId : opt nat64, headers: opt vec (text, text) };
  Provider : nat64;
  Custom : record { url : text; headers : vec (text, text) };
};

request : (
  source : JsonRpcSource,
  jsonRequest : text,
  maxResponseBytes : nat64
) -> (
  Result<text, RpcError>
);
```

I suppose we could directly include this in the documentation as a reference point for various use cases.

Reply:
Added this to the doc